tests/select_for_update/tests.py

```diff
@@ -179,6 +180,19 @@ def test_for_update_requires_transaction_only_in_execution(self):
         with self.assertRaises(transaction.TransactionManagementError):
             list(people)
 
+    @skipUnlessDBFeature('supports_select_for_update_with_limit')
+    def test_select_for_update_with_limit(self):
+        other = Person.objects.create(name='Grappeli')
+        with transaction.atomic():
+            qs = list(Person.objects.all().order_by('pk').select_for_update()[1:2])
+            self.assertEqual(qs[0], other)
+
+    @skipIfDBFeature('supports_select_for_update_with_limit')
+    def test_unsupported_select_for_update_with_limit(self):
+        with self.assertRaises(NotSupportedError):
```

Comment:
Heh. This line was an `assertRaisesMessage` before, I recommended that it be changed to assertRaises to make the test less prone to break on trivial code changes. The message's content is essentially just "Not supported" anyway, so I think we should leave it this way.